kore/src/Kore/Exec.hs

```diff
@@ -220,20 +221,21 @@ execGetExitCode
         , MonadSMT smt
         , MonadUnliftIO smt
         )
-    => VerifiedModule StepperAttributes Attribute.Axiom
+    => Limit Natural
+    -> VerifiedModule StepperAttributes Attribute.Axiom
     -- ^ The main module
     -> ([Rewrite] -> [Strategy (Prim Rewrite)])
     -- ^ The strategy to use for execution; see examples in "Kore.Step.Step"
     -> TermLike Variable
     -- ^ The final pattern (top cell) to extract the exit code
     -> smt ExitCode
-execGetExitCode indexedModule strategy' finalTerm =
+execGetExitCode breadthlimit indexedModule strategy' finalTerm =
     case resolveInternalSymbol indexedModule $ noLocationId "LblgetExitCode" of
         Nothing -> return ExitSuccess
         Just mkExitCodeSymbol -> do
             exitCodePattern <-
                 -- TODO (thomas.tuegel): Run in original execution context.
-                exec indexedModule strategy'
+                exec breadthlimit indexedModule strategy'
```

Comment:
The `breadthLimit` here should be hardcoded to `1`. Execution must not branch when we are choosing the exit code.